views/partials/jira-configuration-table.hbs

```diff
@@ -0,0 +1,185 @@
+{{#if successfulConnections}}
+  <div class="jiraConfiguration__tableContainer">
+    <table class="jiraConfiguration__table aui">
+      <thead class="jiraConfiguration__table__head">
+      <tr class="jiraConfiguration__table__head_row">
+        <th class="jiraConfiguration__table__head__title">Connected organization</th>
+        <th class="jiraConfiguration__table__head__title">Repository access</th>
+        <th class="jiraConfiguration__table__head__title">Backfill status
+          <span class="jiraConfiguration__table__infoIcon aui-icon aui-iconfont-info-filled"
+                id="sync-status-modal-btn">
+            Information
+          </span>
+        </th>
+        <th class="jiraConfiguration__table__head__title">Settings</th>
+      </tr>
+      </thead>
+
+      <tbody class="jiraConfiguration__table__body">
+      {{#each successfulConnections as |connection|}}
+        <tr class="jiraConfiguration__table__row">
+          {{! Organization }}
+          <td class="jiraConfiguration__table__cell">
+            <img
+                class="jiraConfiguration__table__cell__avatar"
```

Comment:
The avatar needs to be hidden for all server accounts. We have to do an investigation into our content security policy this week to solve an issue with the auto app creation. Changes we make may end up resolving issues we're facing with GHE avatars but, for now, I say we just hide them.

Reply:
Avatars hidden for GHE servers